rakelib/github.rake

```diff
@@ -0,0 +1,52 @@
+require 'octokit'
+
+def repo_slug
+  url_parts = `git remote get-url origin`.chomp.split(%r{/|:})
+  last_two_parts = url_parts[-2..-1].join('/')
+  last_two_parts.gsub(/\.git$/, '')
+end
+
+def github_client
+  Octokit::Client.new(:netrc => true)
+end
+
+namespace :github do
+  # rake github:create_release_pr[version]
+  task :create_release_pr, [:version] do |_, args|
+    version = args[:version]
+    branch = release_branch(version)
+
+    title = "Release #{version}"
+    body = <<~BODY
+      This PR prepares the release for version #{version}.
+  
+      Once the PR is merged into master, run `bundle exec rake release:finish` to tag and push to trunk.
+    BODY
+
+    header "Opening PR"
+    res = github_client.create_pull_request(repo_slug, "master", branch, title, body)
+    info "Pull request created: #{res['html_url']}"
+  end
+
+  # rake github:tag
+  task :tag do
+    tag = current_pod_version
+    sh("git", "tag", tag)
+    sh("git", "push", origin, tag)
+  end
+
+  # rake github:create_release
+  task :create_release do
+    tag_name = current_pod_version
+    title = "Release #{args[:version]}"
```

Comment:
Whoops, 2 errors there: (1) there's no `args[:version]` for this task anymore since the recent refactoring, and (2) our GitHub releases are usually only named after the name of the tag without "Release" prefix in the name (that was a bad copy/paste from line 19)

```suggestion
    title = tag_name
```